### DISCUSSION TITLE
Co-registration as batch work

### DISCUSSION THREAD

**Discussion by xav69ier** [CBICA/CaPTk#1413] [Q&A] [answered]
Hi There I plan to co-register a number of image sets stored in different folders, all images are named equal in their respective different folder. I would like to command CaPTk to do all co-registrations as a batch job, and use the co-registered images to populate other folder structure with only those co-registered images, is that possible from the nice GUI?  Many thanks for your feedback: Xavier

**Comment by AlexanderGetka-cbica** [comment, marked as answer]
Hi Xavier,

We're considering this as a feature for the GUI in a future release -- thank you for the suggestion!
Unfortunately in the meantime, the only way to do batch co-registration would be from the command line using a for-loop.
Please let us know if you need help with this.

Thanks,
Alex

**Reply by xav69ier** [reply]
HI ALex 
thanks for the answer - yes I used the command line and the for loop in a shell script - best x
